result/neighborsv6.yaml

```diff
@@ -0,0 +1,43 @@
+openapi: 3.1.0
+
+info:
+  title: IPv6 Neighbors models
+  version: ^0.0.0
+
+components:
+  schemas:
+    Neighborsv6.States.Request:
+      description: >-
+        The request to retrieve IPv6 Neighbor state (NDISC cache entries) for device network interface(s).
```

Comment:
state -> state(s) ?

Reply:
Initially, it was "states". Ankur suggested changing it to "state"

Reply:
IMO a neighbor can only have one state at any given time, it can't have more than one, hence 'state' would be more appropriate than 'state(s)'.